planutils/packages/optic/manifest.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "The OPTIC temporal planner",
+    "description": "The OPTIC temporal planner (see https://github.com/roveri-marco/optic)",
+    "install-size": "297MB",
```

Comment:
Bit of a heavy hitter! Already put in the tips/tricks from the wiki on making it small? We've other planners that clock in at this size, so I'm assuming so, but just want to make sure.